pkg/operator/operator.go

```diff
@@ -47,25 +47,25 @@ func ValidateRBACAfterDisable(cs *clients.Clients, rnames utils.ResourceNames) {
 }
 
 func ValidatePipelineDeployments(cs *clients.Clients, rnames utils.ResourceNames) {
-	EnsureTektonPipelineExists(cs.TektonPipeline(), rnames)
+	EnsureTektonPipelineExists(cs.TektonPipeline(), rnames) //nolint
```

Comment:
The `EnsureTektonPipelineExists` function returns an error, isn't it good to verify if the error is present and fail the test if the error is present?

Reply:
will add

Reply:
Done